Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenWindowControllerIOS.mm

```diff
@@ -1562,6 +1560,13 @@ - (void)_configureSpatialFullScreenTransition
     });
 }
 
+- (void)_setOrnamentsVisabilityForWindow:(UIWindow *)window visible:(BOOL)visible
+{
+    MRUIPlatterOrnamentManager *ornamentManager = window.windowScene._mrui_platterOrnamentManager;
+    for (MRUIPlatterOrnament *ornament in ornamentManager.ornaments)
+        ornament.viewController.view.window.alpha = visible ? 1.0 : 0.0;
```

Comment:
Should we remember which ones we touched on the way in, and only un-touch those on the way out? What if the app added a hidden one in the meantime? (probably not a problem, "added a visible one" would be more likely but not a problem with your existing code).

Reply:
Ya, I suppose I can just do what we already do with ornamentDepths

Reply:
I would reuse the `ornamentDepths` map. Also, passing in the window is unnecessary here, since both windows are in the same scene.